lib/travis/testing/stubs.rb

```diff
@@ -266,6 +266,7 @@ def stub_user(attributes = {})
           tokens: [stub('token', token: 'token')],
           github_scopes: Travis.config.oauth2.scopes.to_s.split(','),
           created_at: Time.now.utc - 7200,
+          first_logged_in_at: Time.now.utc - 14400,
```

Comment:
why the minus 4 hours?

Reply:
That’s what I’m going to test against, it’s random.